src/cmd/cmd.zig

```diff
@@ -452,18 +481,49 @@ fn validator() !void {
     shred_collector_manager.join();
 }
 
+fn validateSnapshot() !void {
+    const allocator = gpa_allocator;
+    const app_base = try AppBase.init(allocator);
+
+    const snapshot_dir_str = config.current.accounts_db.snapshot_dir;
+    var snapshot_dir = try std.fs.cwd().makeOpenPath(snapshot_dir_str, .{});
+    defer snapshot_dir.close();
+
+    const snapshot_result = try loadSnapshot(
+        allocator,
+        app_base.logger,
+        null,
+        true,
+        false,
+    );
+    defer snapshot_result.deinit();
+}
```

Comment:
this is a new command which allows us to validate an existing snapshot in one command